pkg/util/k8s/name.go

```diff
@@ -55,3 +55,12 @@ func ParseStatefulSetName(name string) (statefulSetName string, index int, err e
 	statefulSetName = strings.Join(splittedName[:len(splittedName)-1], "-")
 	return
 }
+
+// GetServiceDNSNames returns the DNS names that can be used to access the given Service.
+func GetServiceDNSNames(namespace, serviceName string) []string {
+	antreaServerName := serviceName + "." + namespace + ".svc"
+	// TODO: Although antrea-agent and kube-aggregator only verify the server name "antrea.<Namespace>.svc",
+	// We should add the whole FQDN "antrea.<Namespace>.svc.<Cluster Domain>" as an alternate DNS name when
+	// other clients need to access it directly with that name.
+	return []string{antreaServerName}
+}
```

Comment:
The location in this package and the name of the function make it sound like it is "generic", but then the variable name `antreaServerName` and the TODO comment kind of make it Antrea-specific.
It's kind of a minor comment though. Maybe we could just rename the variable to something more appropriate.

Reply:
Good catch. I missed updating it when moving the function. 
Updated.